src/graphics/graphics-device.js

```diff
@@ -224,12 +224,18 @@ class GraphicsDevice extends EventHandler {
     canvas;
 
     /**
-     * The highest shader precision supported by this graphics device. Can be 'hiphp', 'mediump' or
-     * 'lowp'.
+     * The WebGL context managed by this graphics device.
      *
-     * @type {string}
+     * @type {WebGLRenderingContext}
```

Comment:
could we mark it with `@ignore`? Ideally we don't expose it publicly I think, as I'm hoping to make most of public interface here private due to WebGPU.

Reply:
Sure, no problem. Does that mean you also want to hide `webgl2`?

Reply:
Perhaps even `webgl2`, even though I don't mind this one, it's just a bool in this case, so does not expose direct access to context

Reply:
The `webgl2` property is currently used in the transform feedback and grab pass examples. It feels to me like there should be a better way to test for the presence of transform feedback than by testing that property. I'm less sure about the shader in the grab pass example. Anyway, for now, I'll hide that too again.